### PR TITLE
Render notebook fields with markdown on content#show pages

### PR DESCRIPTION
**Pull request by drusepth**
Renders field contents with markdown instead of with Ruby's `simple_format`. Features:
- Adds spacing between paragraphs
- Use descending-level headers with `#Title` `##Subtitle` `###Sub-subtitle`, etc
- **Bold** text
- _Italic text_
- _Underlined text_ (with _ on either side -- github makes it italic)
- ~~strikethrough~~ text
- ^superscript text
- autolinking URLs
- markdown-style URL linking e.g. [google](http://www.google.com)
- php-style table formatting
- php-style footnotes
- strips out user-supplied HTML (e.g. script tags, etc)

We still want to do content-linking (e.g. [[Bob]]), but I think getting markdown formatting in place is a good start. This still needs some formatting help visible to users explaining markdown (or at least a link somewhere) before it can go out.

### CODE REVIEW

File: app/assets/stylesheets/content.css.scss [hunk 1]
```diff
@@ -29,4 +29,22 @@ p.long-form {
...
+
+    li {
+      list-style-type: initial !important;
+    }
+  }
+
+  ol {
+    padding: 0;
+  }
 }
```
Comment (Cantido) [inline app/assets/stylesheets/content.css.scss]: Missing a newline! Install the [editorconfig](http://editorconfig.org/) plugin into whatever editor you use, then these will be automatically taken care of.

File: app/views/content/_show.html.erb [hunk 2]
```diff
@@ -28,6 +28,22 @@ set_meta_tags title: content.name, description: content.description
             <li class="tab col s3 <%= "disabled" unless data[:attributes].any? { |m| content.send(m).present? } %>"><a href="#<%= category %>"><%= category.to_s.humanize %></a></li>
           <% end %>
         </ul>
+        <%
+          # TODO: Move this to some app-wide initializer
```
Comment (Cantido) [inline app/views/content/_show.html.erb]: Would it be better to make an issue card for this? We'd lose track of it like this.

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -44,6 +44,7 @@
 <div class="navbar-fixed">
   <nav class="light-blue">
     <div class="nav-wrapper">
+
```
Comment (Cantido) [inline app/views/layouts/_navbar.html.erb]: :question:

File: app/assets/stylesheets/content.css.scss (hunk 1 above)
Comment (drusepth) [inline app/assets/stylesheets/content.css.scss]: You got it! :)

File: app/views/content/_show.html.erb (hunk 2 above)
Comment (drusepth) [inline app/views/content/_show.html.erb]: Yep, probably. I'll make one and delete this comment.

**Comment by drusepth** [comment]
Thanks for the feedback. Made the changes and merged this into the `content-expansion` branch to release at the same time.

**Pull request closed without merging**